pkgs/http/pubspec.yaml

```diff
@@ -20,3 +20,20 @@ dev_dependencies:
   shelf: ^1.1.0
   stream_channel: ^2.1.0
   test: ^1.16.0
+
+dependency_overrides:
```

Comment:
This this temporary? Should we release with these overrides?

Reply:
They don't affect downstream users, and are required right now for the tests to pass until https://github.com/dart-lang/test/pull/2144 lands (sometime next week hopefully).

You could also put these in a `pubspec_overrides.yaml` file if you prefer, which might be a bit cleaner?

Reply:
If the intention is to not affect downstream users, shouldn't we use `dev_dependency_overrides`?

Reply:
NO! Oops

Reply:
> NO! Oops

Does that mean that you are changing something here?

Reply:
There is only `dependency_overrides`, and they never have any effect on any package other than the one in which they are written. 

In this case, they are only overrides of dev dependencies, so it is safe to release that way. You almost never would want to release with any dependency overrides of regular deps though (although in the case of cycles it _can_ be necessary).

Or, wait until the new version of package:test is released before landing this.

Reply:
dropped